interpreter/interpreter_import.go

```diff
@@ -32,13 +32,13 @@ func (interpreter *Interpreter) VisitImportDeclaration(declaration *ast.ImportDe
 	resolvedLocations := interpreter.Program.Elaboration.ImportDeclarationsResolvedLocations(declaration)
 
 	for _, resolvedLocation := range resolvedLocations {
-		interpreter.importResolvedLocation(resolvedLocation)
+		interpreter.importResolvedLocation(resolvedLocation, &declaration.Aliases)
 	}
 
 	return nil
 }
 
-func (interpreter *Interpreter) importResolvedLocation(resolvedLocation sema.ResolvedLocation) {
+func (interpreter *Interpreter) importResolvedLocation(resolvedLocation sema.ResolvedLocation, aliases *map[string]string) {
```

Comment:
Go `map`s already have reference semantics (a value with a pointer), so there is no need to pass it by pointer. Maybe also chop down long parameter lists over multiple lines:

```suggestion
func (interpreter *Interpreter) importResolvedLocation(
	resolvedLocation sema.ResolvedLocation,
	aliases map[string]string,
) {
```

(also need to adjust the call-site above, and the dereference below)